src/Workspaces/Core/Portable/NamingStyles/NamingStyle.cs

```diff
@@ -346,7 +346,7 @@ private string CreateCompliantNameReusingPartialPrefixesAndSuffixes(string name)
             return FinishFixingName(name);
         }
 
-        private static string StripCommonPrefixes(string name, out string prefix)
+        internal static string StripCommonPrefixes(string name, out string prefix)
```

Comment:
can be public. generally we don't have internal members (except inside public types).

Reply:
:memo: This isn't a rule for this project. Both options are available for internal types.

Reply:
It's not a rule.  but it's the general pattern we follow for the majority of code in the codebase.

> f you would like this to be a rule, please file it as an analyzer request in dotnet/roslyn-analyzers and/or itemizing it in #31140 rather than adding it to reviews.

i think it would be better to flip things around.  If you'd prefer to not have code consistency brought up in reviews, then it would be good to make the rule and demonstrate that it's in effect.
'
That switches the burden around.  That way we keep the code consistent by default, and if we want to lighten teh PR process, people can put in the legwork.  If we go the way you mentioned, it will end up making it so that we have all these requests (which likely won't go anywhere) and in teh meantime code consistency slides down.